frigate/restream.py

```diff
@@ -12,7 +12,7 @@
 
 def get_manual_go2rtc_stream(camera_url: str) -> str:
     """Get a manual stream for go2rtc."""
-    return f"exec: /usr/lib/btbn-ffmpeg/bin/ffmpeg -i {camera_url} -c:v copy -c:a libopus -rtsp_transport tcp -f rtsp {{output}}"
+    return f"ffmpeg:{camera_url}#video=copy#audio=aac#audio=opus"
```

Comment:
@NickM-27 I'm speaking about this. I was thinking on something like:

```suggestion
    return f"ffmpeg:{camera_url}#raw={hwaccel_args}#video=copy#audio=aac#audio=opus"
```